app/src/main/java/com/kickstarter/libs/AnalyticEvents.kt

```diff
@@ -666,6 +667,16 @@ class AnalyticEvents(trackingClients: List<TrackingClientType?>) {
         client.track(SEARCH_BUTTON_CLICKED)
     }
 
+    /**
+     * Sends data associated with the search CTA click event to the client.
+     *
+     * @param discoveryParams: The discovery data.
+     */
+    fun trackSearchCTAButtonClicked(discoveryParams: DiscoveryParams) {
+        val props = AnalyticEventsUtils.discoveryParamsProperties(discoveryParams)
```

Comment:
We need to add the property `context_cta` with a value of `search`, as well as the `context_location` with a value of `global_nav`, to the `props` object